neo/public/javascripts/timeseries/controllers.js

```diff
@@ -74,14 +77,15 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
               .html(minDate.getFullYear()+"-"+(minDate.getMonth()+1)+"-"+minDate.getDate());
 
             timeSeries
-              .renderArea(true)
```

Comment:
just curious, why should we remove this line? though I'm not sure what it is doing?